modules/core/src/main/scala/org/tesselation/domain/cluster/storage/AddressStorage.scala

```diff
@@ -1,12 +1,21 @@
 package org.tesselation.domain.cluster.storage
 
+import org.tesselation.infrastructure.db.schema.StoredAddress
```

Comment:
you cannot import `infrastructure` into `domain`. if so, `StoredAddress` should be moved and/or renamed

Reply:
Fixed. I'm now passing `(Address, Balance)` tuple.